package.json

```diff
@@ -21,9 +21,10 @@
   },
   "peerDependencies": {},
   "dependencies": {
+    "@tweenjs/tween.js": "^17.2.0",
     "@vimeo/player": "^1.0.6",
     "async": "^2.3.0",
-    "audio5": "^0.1.10",
```

Comment:
The updates have been merged into the npm version of the library.

I'll flip this to the new release version via npm